as2/6.3/modules/ROOT/pages/as2-connector-examples.adoc

```diff
@@ -0,0 +1,223 @@
+|===
++
+. Configure a connection to the partner if the MDN request is to be sent asynchronously.
+. If you want to configure other connection details (such as TLS, proxy, and timeouts) do that now.
+
+== Sending MDNs to Multiple Partners in Partner Manager
+
+The following example shows how to send MDNs to multiple partners. This example contains two flows:
+
+* The first flow listens for AS2 messages on path `/forward-as2` and forwards them to `/forward-van/as2-receive`.
+* The second flow sends
+
+=== Listening for AS2 Messages
+
+The following flow listens for AS2 messages:
+
+image::as2-connector-example-send-mdn-2.png[Flow for receiving and forwarding AS2 messages]
+
+The XML for this example is:
+
+[source,xml,linenums]
+----
+
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="UTF-8"?>
+
+<mule xmlns:ee="http://www.mulesoft.org/schema/mule/ee/core" xmlns:tls="http://www.mulesoft.org/schema/mule/tls"
+	xmlns:http="http://www.mulesoft.org/schema/mule/http"
+	xmlns:as2-mule4="http://www.mulesoft.org/schema/mule/as2-mule4" xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:doc="http://www.mulesoft.org/schema/mule/documentation" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+http://www.mulesoft.org/schema/mule/as2-mule4 http://www.mulesoft.org/schema/mule/as2-mule4/current/mule-as2-mule4.xsd
+http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+http://www.mulesoft.org/schema/mule/tls http://www.mulesoft.org/schema/mule/tls/current/mule-tls.xsd
+http://www.mulesoft.org/schema/mule/ee/core http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd">
+	<http:listener-config name="HTTP_Listener_config" doc:name="HTTP Listener config" doc:id="35d8e9ef-c8f9-4364-aa90-957de030b9b4" basePath="b2b" >
+		<http:listener-connection protocol="HTTPS" host="0.0.0.0" port="${https.port}" >
+			<tls:context >
+				<tls:key-store type="jks" path="rootca.jks" alias="rootca" keyPassword="test" password="test" />
+			</tls:context>
+		</http:listener-connection>
+	</http:listener-config>
+
+	<configuration-properties doc:name="Configuration properties" doc:id="b0baf25d-17ef-4d10-bb24-2c5b25e6289c" file="app.properties" />
+	<as2-mule4:listener-config name="AS2_Connector_Listener_ForwardVAN" doc:name="AS2 Connector Listener config" doc:id="59fd60b5-92f4-4532-9b03-41be1e8c27bc" httpListenerConfig="HTTP_Listener_config" securityLevel="SIGNED_ENCRYPTED" >
+		<as2-mule4:self-config as2Name="forward-van" x509Alias="forward-van" email="support@forward-van.com" />
+		<as2-mule4:partner-config as2Name="myth-as2" x509Alias="myth-as2" email="support@mythical.com" />
+		<as2-mule4:key-store-config keystorePassword="test" keystorePath="as2/forward-van.p12" privateKeyPassword="test" />
+	</as2-mule4:listener-config>
```

Comment:
Please remove this part. It was in the app for some POC. It need not be in the docs.